content/en/security/cloud_security_management/setup/agentless_scanning/quick_start.md

```diff
@@ -27,8 +27,8 @@ For existing users who want to add a new AWS account or enable Agentless Scannin
 1. Click **Start Using Cloud Security Management** to continue.
 1. Select the AWS region where you want to create the CloudFormation stack.
 1. Select an API key that is already configured for Remote Configuration. If the API key you select does not have Remote Configuration enabled, Remote Configuration is automatically enabled for that key upon selection.
-1. Send **AWS Logs to Datadog** and **Enable Cloud Security Management** are automatically selected by default. Leave the selections as is.
-1. In the **Agentless Scanning** section, toggle **Host Vulnerability Scanning**, **Container Vulnerability Scanning**, **Lambda Vulnerability Scanning**, and **Data Security Scanning** to the on position.
+1. **Send AWS Logs to Datadog** and **Detect security issues** are automatically selected by default. Leave the selections as is.
+1. The **Enable Vulnerability Management (Host, Container and Lambda)** switch is also enabled by default. Leave this selection as is.
```

Comment:
```suggestion
1. The **Enable Vulnerability Management (Host, Container, and Lambda)** switch is also enabled by default. Leave this selection as it is.
```

Reply:
As an alternative to adding it, you could also hyphenate "as is" to make it function as an adjective

Reply:
Thanks. That's a good callout. I think I'll go ahead and hyphenate the word "as-is".